docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - ess-local
     environment:
       DATABASE_URI: postgresql+psycopg2://${DB_POSTGRES_USER:-ess}:${DB_POSTGRES_PASSWORD:-ess}@db:5432/${DB_POSTGRES_DB:-ess}
-      SOLR_URL: http://149.156.182.2:8983/solr/
+      SOLR_URL: http://149.156.182.69:8983/solr/
```

Comment:
I am not convinced that we should connect to the production solr instance by default. Its risky. I would prefer to keep beta instance as it was